### PR TITLE
Added regolith-session-common binary package

### PR DESCRIPTION
**Pull request by SoumyaRanjanPatnaik**
`regolith-session-flashback` and `regolith-session-sway` have several common files. Trying to install both packages end in an error from `dpkg` saying
```
trying to overwrite '/usr/bin/regolith-diagnostic', which is also in package regolith-session-flashback
```
This PR moves common files required by both sessions into the package `regolith-session-common` in order to fix this issue.

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -44,8 +46,15 @@ Depends: ${misc:Depends},
     trawldb,
     wl-clipboard,
     xwayland
-Provides: regolith-desktop-session
```
Comment (kgilmer) [inline debian/control:47]: Why did you remove this?  (This is a virtual package specifier.  It is referenced [here](https://github.com/regolith-linux/regolith-desktop/blob/debian/debian/control#L15) as a way to say "install the flashback session or another package that provides the same virtual package)

**Review by kgilmer — CHANGES_REQUESTED** [review]

File: debian/control (hunk 1 above)
Comment (SoumyaRanjanPatnaik) [reply debian/control:47]: Actually, providing the `regoliith-desktop-session` virtual package causes `regolith-session-flashback` and `regolith-session-sway` to conflict. So, both packages cannot be installed together without removing the line.

Comment (kgilmer) [reply debian/control:47]: Huh, not sure how that would be true as I don't find a `Conflicts` cause but will merge for now and see if I can repro it

**Review by kgilmer — APPROVED** [review]
